Syntaxes/SCSS.sublime-syntax

```diff
@@ -549,6 +542,20 @@ contexts:
     - meta_scope: meta.at-rule.return.scss
     - include: values
 
+###[ SCSS EXTEND AT-RULE ]#####################################################
+
+  scss-at-extend:
+    - match: (@)extend{{break}}
+      scope: keyword.control.directive.extend.scss
+      captures:
+        1: punctuation.definition.keyword.scss
+      push: scss-at-extend-body
+
+  scss-at-extend-body:
```

Comment:
TIL I learned that that creates a named context and that that help _a ton_. I feel like such a rookie. Thanks for spending all this time on this syntax, pretty excited about where this is going 👍🏻

Reply:
And it definitely makes it easier to figure out "where did this thing come from" 👍🏻